apps/style/common.scss

```diff
@@ -1201,6 +1201,22 @@ $workspace-header-button-margin: ($workspace-headers-height - $workspace-header-
   }
 }
 
+.show-code {
+  summary {
+    font-size: 18px;
+    line-height: 20px;
+    color: #5b6770;
+    font-family: "Gotham 5r", sans-serif;
+    font-weight: normal;
+    outline: none;
+    padding: 5px;
+  }
+
+  .generated-code-container {
+    margin: 25px 10px 25px -80px;
+  }
+}
+
```

Comment:
One issue with inline styles (both for this and for the other Code component) is that it makes it difficult to situationally override styles. In particular, the minecraft styles have a couple overrides apiece for all of these components.